config/v1/types_node.go

```diff
@@ -48,7 +48,13 @@ type NodeSpec struct {
 	WorkerLatencyProfile WorkerLatencyProfileType `json:"workerLatencyProfile,omitempty"`
 }
 
-type NodeStatus struct{}
+type NodeStatus struct {
+	// Conditions contain the details and the current state of the nodes.config object
```

Comment:
Use the json tag version of the field name to start the comment.

Do you have any known condition types you could enumerate here?

Reply:
Addressed, There are no known condition types to enumerate as of now.